source/index.ts

```diff
@@ -162,6 +162,8 @@ namespace is { // tslint:disable-line:no-namespace
 	export const arrayBuffer = isObjectOfType(TypeName.ArrayBuffer);
 	export const sharedArrayBuffer = isObjectOfType(TypeName.SharedArrayBuffer);
 
+	export const directInstanceOf = (instance: any, klass: any) => object(instance) && object(klass) && Object.getPrototypeOf(instance) === klass.prototype;
```

Comment:
ES5 will throw a `TypeError` if `instance` isn't an object. ES2015 will coerced it to an object, and not throw an error. To support the lowest common denominator, I've required `instance` to be an `object` here to cover our bases and protect against an exception. I could be persuaded otherwise though if there is other thoughts on this.

Reply:
[ref](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/getPrototypeOf)

Reply:
Can you add a `TODO` comment about removing the object wrappers when we target only ES2015?